bin/cron/delete_old_ai_chat_data

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env ruby
+require 'cdo/chat_client'
+require_relative 'only_one'
+abort 'Script already running' unless only_one_running?(__FILE__)
+
+require_relative '../../dashboard/config/environment'
+
+# We want to avoid storing student chat messages any longer than
+# is necessary for analysis to prevent storing excess student PII. This deletes
+# any AichatEvent or AichatRequest older than 90 days.
+# 
+# Additionally, we delete any AichatSession older than 30 days. AichatSession is
+# no longer in use and we plan to fully delete the table.
+# See https://codedotorg.atlassian.net/browse/LABS-976 for details.
+#
+# Log the number of records deleted for each table to Slack, #cron-watcher-aichat-deletion
+# so we can easily monitor the deletion process.
+def main
+  thirty_days_ago = Time.now - 30.days
+  ninety_days_ago = Time.now - 90.days
+
+  destroyed_sessions = AichatSession.where('created_at < ?', thirty_days_ago).destroy_all
```

Comment:
Could we hold off on deleting data from this table for now? Asked @samantha-code about this [over Slack](https://codedotorg.slack.com/archives/C06UT1E2796/p1723659639734519) and she would like to keep the student chat data for now.

Reply:
@fisher-alice @sanchitmalhotra126 I updated to keep student chat data from the pilot, and updated https://codedotorg.atlassian.net/browse/LABS-976 with a reminder to revisit this. Presumably when Product is ready to delete the pilot data, we can both update the cron job and delete `AiSessions`. How does that sound?